### PR TITLE
Update Docker Scout overview page

### PR DESCRIPTION
**Pull request by ChrisChinchilla**
### Proposed changes

Updates Docker Scout overview page to add higher level concepts, clearer explanation of where to find out more and some video content.

### CODE REVIEW

File: scout/index.md [hunk 1]
```diff
@@ -16,62 +16,60 @@ redirect_from:
 
 {% include scout-early-access.md %}
 
-Docker Scout is a collection of software supply chain features that appear
-throughout Docker user interfaces and the command line interface (CLI). These features provide detailed
-insights into the composition and security of container images.
+Container images are often built from layers of other container images and
```
Comment (usha-mandya) [inline scout/index.md:19]: I think the current introductory section still makes sense as it clearly states what Scout is all about. Can we retain the intro section and then add this?

File: scout/index.md [hunk 2]
```diff
@@ -16,62 +16,60 @@ redirect_from:
...
-Docker Scout analyzes image contents and generates a detailed report of
-packages and vulnerabilities that it detects. Docker Scout can also help provide
-you with suggestions for how you can remediate issues discovered by the image
-analysis.
+Docker Scout can proactively help you find and fix these vulnerabilities,
+helping you create a more secure software supply chain.
 
-The [image details view](./image-details-view.md) in Docker Desktop and the tag
-details pages on Docker Hub are both powered by Docker Scout.
+It does this by analyzing your images and creating a full inventory of the
```
Comment (usha-mandya) [inline scout/index.md:26]: Suggest moving this section to the paragraph above or clarify what we mean by "It" in the "It does this.." and "It then correlates".

File: scout/index.md [hunk 3]
```diff
@@ -16,62 +16,60 @@ redirect_from:
...
-`docker scout`
-[plugin for Docker CLI](../engine/reference/commandline/scout_cves.md).
+You can use Docker Scout in Docker Desktop, Docker Hub, the Docker CLI, and in
+the Docker Scout Dashboard. If you host your images in JFrog Artifactory, you
+can also use Docker Scout to analyze your images there.
 
-There's also a [Dashboard](https://scout.docker.com){: target="\_blank"
-rel="noopener" } that you can use to explore additional information about
-images, packages, and CVEs.
+<div style="position: relative; padding-bottom: 64.86486486486486%; height: 0;"><iframe src="https://www.loom.com/embed/e066986569924555a2546139f5f61349?sid=6e29be62-78ba-4aa7-a1f6-15f96c37d916" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;"></iframe></div>
```
Comment (usha-mandya) [inline scout/index.md:35]: Can we add a brief description about what the video is all about?

File: scout/index.md [hunk 4]
```diff
@@ -80,11 +78,31 @@ $ curl -fsSL https://raw.githubusercontent.com/docker/scout-cli/main/install.sh
...
-Always examine scripts downloaded from the internet before running them locally. Before installing, make yourself familiar with potential risks and limitations of the convenience script:
-
-> **Tip**
+> **Note**
 >
-> If you want to install the plugin manually, you can find full instructions in the [plugin's repository](https://github.com/docker/scout-cli).
-{: .tip }
+> Always examine scripts downloaded from the internet before running them locally.
+> Before installing, make yourself familiar with potential risks and limitations
+> of the convenience script
```
Comment (usha-mandya) [inline scout/index.md:85]: nit

```suggestion
> of the convenience script.
```

File: scout/index.md [hunk 5]
```diff
@@ -80,11 +78,31 @@ $ curl -fsSL https://raw.githubusercontent.com/docker/scout-cli/main/install.sh
...
+## JFrog Artifactory integration
+
+Users of JFrog Artifactory, or JFrog Container Registry, can integrate Docker
+Scout to enable automatic analysis of images locally and remotely. For more
+information, see [Artifactory integration](./artifactory.md).
+
 
-The plugin is also available as [a container image](https://hub.docker.com/r/docker/scout-cli) and as [a GitHub action](https://github.com/docker/scout-action)
\ No newline at end of file
+<div style="position: relative; padding-bottom: 64.86486486486486%; height: 0;"><iframe src="https://www.loom.com/embed/a6fb14ede0a94d0d984edf6cf16604e0?sid=ba34f694-32a6-4b74-b3f8-9cc6b80ef66f" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;"></iframe></div>
```
Comment (usha-mandya) [inline scout/index.md:108]: Could you add a description about the video?

Comment (ChrisChinchilla) [reply scout/index.md:111]: Done, this one doesn't completely fit here, but the team want to host it somewhere and when the onboarding guide is finished, there will be a better place to put it.

File: scout/index.md (hunk 1 above)
Comment (ChrisChinchilla) [reply scout/index.md:19]: Yes, this change was based on feedback from a few of the team.

**Review by aevesdocker — APPROVED** [review]
🚀

**Pull request merged**